<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="HP" failures="0" tests="3" name="mypackage.NewTest" time="0.240" errors="1" timestamp="2020-03-08T15:48:38 IST" skipped="0">
  <testcase classname="mypackage.NewTest" name="testcase2" time="0.005"/>
  <testcase classname="mypackage.NewTest" name="testcase3" time="0.004"/>
  <testcase classname="mypackage.NewTest" name="testcase1" time="0.231">
    <error message="Method mypackage.NewTest.testcase1() didn&#039;t finish within the time-out 100" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method mypackage.NewTest.testcase1() didn't finish within the time-out 100
]]>
    </error>
  </testcase> <!-- testcase1 -->
</testsuite> <!-- mypackage.NewTest -->
